packages/synapse-interface/pages/bridge/BridgeWatcher/index.tsx

```diff
@@ -22,76 +23,221 @@ import {
   generateBridgeTx,
   checkTxIn,
 } from '@utils/bridgeWatcher'
+import { SynapseSDK } from '@/../sdk-router/dist'
```

Comment:
this is just for local testing?

Reply:
ya think this might be throwing the following error: 

what's the most updated version the sdk-router should be on right now?
![Screenshot 2023-06-28 at 11 08 45 PM](https://github.com/synapsecns/sanguine/assets/57741810/be8715fb-bfeb-45c2-a49e-8e05d5756785)

Reply:
fixing now, sry